### DISCUSSION TITLE
Code in "Plotting with Geoplot and GeoPandas" in "Examples Gallery" not working and .ipynb not accessible via the main branch

### DISCUSSION THREAD

**Discussion by timmhopp** [geopandas/geopandas#3274] [General]
The notebook in the CartoPy & GeoPandas section isn't working and no longer available on the main branch of GeoPandas.

https://geopandas.org/en/stable/gallery/plotting_with_geoplot.html

**Comment by martinfleis** [comment]
That is correct. The geoplot library is no longer maintained and the example has been removed. It will disappear from the documentation with the next release.

**Reply by timmhopp** [reply]
Alright, thank you!
